src/ts/ConvivaAnalyticsTracker.ts

```diff
@@ -621,15 +621,19 @@ export class ConvivaAnalyticsTracker {
       PlayerEvent.Play,
```

Comment:
I can't comment on line 614, but this method is not automatically called for every event. This call still needs to be added accordingly. This call should go into the `ConvivaAnalytics.ts` in the corresponding callback.

However, I tried to do that and couldn't make it work to report buffering to the Ad session. It looks like the player emits the `AdBreakStarted` event only when the VAST and Media Files are resolved (I tried throttling both). Therefore, the buffering is counted towards the main content and not toward the ad 🤔 I am not sure if this is by design. I'm afraid this solution needs more thinking and is not as low-hanging as initially thought.

Reply:
Yeah I suppose this won't actually track the delay in the ad manifest loading, but should at least track successfully the VST of the ad 👀